tracer/src/Datadog.Trace/ClrProfiler/AutoInstrumentation/AWS/Lambda/LambdaCommon.cs

```diff
@@ -21,50 +22,42 @@ internal abstract class LambdaCommon
     private const double ServerlessMaxWaitingFlushTime = 3;
     private const string LogLevelEnvName = "DD_LOG_LEVEL";
 
-    internal static Scope CreatePlaceholderScope(Tracer tracer, string traceId, string samplingPriority)
+    internal static Scope CreatePlaceholderScope(Tracer tracer, WebHeaderCollection headers)
```

Comment:
You should use the `WebHeadersCollection` wrapper instead of `WebHeaderCollection`

```suggestion
    internal static Scope CreatePlaceholderScope(Tracer tracer, WebHeadersCollection headers)
```

Reply:
Rider doesn't believe WebHeadersCollection exists. WebHeaderCollection.Wrap() gives me a NameValueHeadersCollection. Does that sound reasonable?